packages/ts-client/src/FishjamClient.ts

```diff
@@ -354,16 +337,17 @@ export class FishjamClient<PeerMetadata, TrackMetadata> extends (EventEmitter as
     this.status = 'initialized';
   }
 
+  private getUrl(url: string) {
+    if (url.endsWith('/')) return url + WEBSOCKET_PATH;
+    return url + '/' + WEBSOCKET_PATH;
```

Comment:
**suggestion:** consider sticking to template literals while working with strings in js as it is a safer operation
```suggestion
    if (url.endsWith('/')) return `${url}${WEBSOCKET_PATH}`
    return `${url}/${WEBSOCKET_PATH}`
```